src/js/media-controller.js

```diff
@@ -710,10 +739,95 @@ class MediaController extends MediaContainer {
     els.splice(index, 1);
   }
 
-  /**
-   * Media Controller should not mimic the HTMLMediaElement API.
-   * @see https://github.com/muxinc/media-chrome/pull/182#issuecomment-1067370339
-   */
+  keyboardShortcutHandler(e) {
+    if (this.contains(e.target) || this.shadowRoot.contains(e.target)) {
+      // if the event's key is already handled by the target, skip keyboard shortcuts
+      // keysUsed is either an attribute or a property.
+      // The attribute is a DOM array and the property is a JS array
+      // In the attribute Space represents the space key and gets convered to ' '
+      const keysUsed = (e.target.getAttribute('keysused')?.split(' ') ?? e.target?.keysUsed ?? [])
```

Comment:
thought(non-blocking): As a followup, we'll need to figure out how this should work for non-nested components based on the current architecture (See code/architecture for `associatedElements` and how event (re)propagation works).

Reply:
yup, this got called out in the discussion #257